src/blocks/Tabs/Tabs.tsx

```diff
@@ -37,15 +38,32 @@ export const TabsBlock = ({
     const isReverse = direction === 'content-media';
     const ref = useRef<HTMLDivElement>(null);
     const mediaWidth = ref?.current?.offsetWidth;
-    const mediaHeight = mediaWidth && getHeight(mediaWidth);
     const captionId = useUniqId();
-
+    const themedMedia = getThemedValue(activeTabData?.media, theme);
+    const mediaVideoHeight = mediaWidth && getHeight(mediaWidth);
+    const setVideoHeight = !themedMedia?.image && !activeTabData?.image;
+    const [minImageHeight, setMinImageHeight] = useState(ref?.current?.offsetHeight);
+    // TODO remove property support activeTabData?.image. Use only activeTabData?.media?.image
     let imageProps;
 
+    const handleImageHeight = useCallback(() => {
+        setMinImageHeight(ref?.current?.offsetHeight);
+    }, []);
```

Comment:
should we add `ref?.current?.offsetHeight` to hook dependencies?

Reply:
ESLINT said that is unnecessary dependency